cassandra-test-image/src/test/java/cassandracluster/AbstractCassandraCluster.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cassandracluster;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import java.net.InetSocketAddress;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.testcontainers.containers.DockerComposeContainer;
+
+public class AbstractCassandraCluster
+{
+    private static DockerComposeContainer<?> composeContainer;
+    protected static String containerIP;
+    protected static CqlSession mySession;
+
+    @BeforeClass
+    public static void setup() throws InterruptedException
+    {
+        Path dockerComposePath = Paths.get("")
+                .toAbsolutePath()
+                .getParent()
+                .resolve("cassandra-test-image/src/main/docker/docker-compose.yml");
+        System.out.println("DockerComposePath que veio foi" + dockerComposePath.toFile());
+        composeContainer = new DockerComposeContainer<>(dockerComposePath.toFile());
+        composeContainer.start();
+
+        System.out.println("Waiting Cassandra Cluster finish to start");
```

Comment:
This sentence sounds a bit weird. Finish to start? Is it like the end of the beginning as Churchill would put it? ;-)